web/packages/teleport/src/Account/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, ButtonSecondary, Flex, Text } from 'design';
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+
+export interface HeaderProps {
+  title: string;
+  description?: string;
+  icon: React.ReactNode;
+  actions: React.ReactNode;
+}
+
+export function Header({ title, description, icon, actions }: HeaderProps) {
+  const theme = useTheme();
+  return (
+    <Flex alignItems="start" gap={3}>
+      {/* lineHeight=0 prevents the icon background from being larger than
+              required by the icon itself. */}
+      <Box
+        bg={theme.colors.interactive.tonal.neutral[0]}
+        lineHeight={0}
+        p={2}
+        borderRadius={3}
+      >
+        {icon}
+      </Box>
+      <Box flex="1">
+        <Text typography="h4">{title}</Text>
```

Comment:
Why `h4`?

Reply:
It was the closest match that I found to the designs. And this being a header, seems to fit the purpose. What's your suggestion?

Reply:
(Actually, to clarify: the design, if you inspect the typography info, calls for "Ubuntu/H2" typography, but the font size corresponds to our h4, so I decided to use that one.)

Reply:
In the future I suggest we don't choose an element based on what font size we get by default, and instead choose the most semantically correct element and use CSS to get the desired font size.

Having an h4 that doesn't fall under an h3 is semantically incorrect and not good for accessibility.